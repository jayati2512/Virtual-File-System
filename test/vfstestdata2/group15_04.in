createvfs myvfs 900
mountvfs myvfs 
makedir / folder1
makedir /abc folder2
makedir /temp
makedir /san a/nt 
deletedir /qwer
movedir /abc /folder1
movedir /abc /dan
addfile /folder2 fil_1.txt /usr/include/stdio.h
removefile /folder/fil_1.txt 
listfile /folder/fil_1.txt /tmp/lisfil.txt
addfile /folder2 fil_1.txt /usr/include/stdio.h
addfile /folder2 fil_2 /usr/include/conio.h
addfile /folder3 fil_1.txt /usr/include/conio.h
listfile /folder2/fil_2 /tmp/include/lisfil2.txt
