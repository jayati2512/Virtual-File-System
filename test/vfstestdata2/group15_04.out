create_vfs_SUCCESS
mount_vfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
makedir_FAILURE INVALID_CHARACTER_IN_DIRNAME
deletedir_FAILURE CANNOT_FIND_SPECIFIED_DIR
movedir_SUCCESS
movedir_FAILURE CANNOT_FIND_SPECIFIED_DESTINATIONDIR
addfile_FAILURE DIRECTORY_DOES_NOT_EXIST
removefile_FAILURE CANNOT_FIND_SPECIFIED_FILE
listfile_FAILURE SOURCE_FILE_PATH_NOT_FOUND
addfile_FAILURE DIRECTORY_DOES_NOT_EXIST
addfile_FAILURE DIRECTORY_DOES_NOT_EXIST
addfile_FAILURE DIRECTORY_DOES_NOT_EXIST
listfile_FAILURE NOT_A_TEXT_FILE
